packages/rocketchat-lg-api-extensions/package.js

```diff
@@ -12,6 +12,7 @@ Package.onUse(function (api) {
   api.use([
     'ecmascript',
     'deepwell:raven@0.3.0',
+    'learnersguild:rocketchat-lg-core@1.0.0',
```

Comment:
We're specifying a minimum dependency version ... it will take 1.0.1, 1.0.2, 1.1.4, etc. It just won't take 0.x.x or 2.x.x.